docs/Upgrading to CSLA 9.md

```diff
@@ -90,6 +90,12 @@ CSLA 9 fixes the issue with transfering binary data between the client and serve
         .UseHttpProxy(o => o.DataPortalUrl = "https://server/api/DataPortal"))));
 ```
 
+## CriteriaBase Obsolete
+
+The `CriteriaBase` class is now obsolete. You should use normal .NET or CSLA-derived types for criteria values instead.
+
+The most direct replacement for a complex criteria class is to subclass `ReadOnlyBase`. This is because criteria classes are typically read-only, and `ReadOnlyBase` provides the necessary functionality.
```

Comment:
Please add the remark that those types must be created through a data portal and not with `new`.
What do we recommend? The POCO serializer or a custom one? Maybe we should mention that.